cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import time
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    """Django management command to wait
+     for the database to be available."""
+
+    def handle(self, *args, **options):
```

Comment:
The `handle` method is correctly implemented to wait for the database connection to be established before proceeding. This aligns with the task requirement to implement a management command that waits for the database to be available.